dragnet/models.py

```diff
@@ -1,54 +0,0 @@
-import gzip
-import os
-import pkgutil
-import warnings
-
-from sklearn import __version__ as sklearn_version
-
-from .compat import PY2, pickle, bytes_io, sklearn_path
-from .blocks import TagCountNoCSSReadabilityBlockifier
-from .content_extraction_model import baseline_model
-from .content_extraction_model import ContentCommentsExtractionModel, SklearnWrapper
-
-
-def _load_pickled_model(fname, compressed='gzip'):
-    model_bytes = pkgutil.get_data(
-        'dragnet', os.path.join('pickled_models', sklearn_path, fname))
-    if compressed == 'gzip':
-        with gzip.GzipFile(fileobj=bytes_io(model_bytes), mode='rb') as f:
-            return pickle.load(f)
-            # if PY3: pickle.load(f, encoding='bytes') ?
-    else:
-        return pickle.loads(model_bytes)
-
-
-try:
-    # make instances
-    content_extractor = _load_pickled_model(
-        'kohlschuetter_weninger_readability_content_model.pickle.gz',
-        compressed='gzip')
-    content_comments_extractor = _load_pickled_model(
-        'kohlschuetter_weninger_readability_content_comments_model.pickle.gz',
-        compressed='gzip')
-
-    # monkey patches
-    content_extractor._blockifier = TagCountNoCSSReadabilityBlockifier
```

Comment:
This is where the `_blockifier` was monkey patched when loading models.

Reply:
I don't know too much about this change, as it was made in @bdewilde's branch, but I would be happy to train a new model that simply uses the `TagCountNoCSSReadabilityBlockifier` if that helps. FWIW, on Python 3 and sklearn 0.19, the model loads with `TagCountReadabilityBlockifier`.

Reply:
I think that's worthwhile to try.  If I remember correctly, the CSS features are pretty slow and don't add much to the overall performance so dropping them and using the `TagCountNoCSSReadabilityBlockifier` usually gives the best accuracy/speed tradeoff.

Reply:
Sounds good, I'll get some new models up in a new PR as soon as I have the chance